src/com/twu/biblioteca/Book.java

```diff
@@ -0,0 +1,35 @@
+package com.twu.biblioteca;
+
+public class Book {
+    private final String title;
+    private final String author;
+    private final int yearPublished;
```

Comment:
Nice - final attributes mean the Book objects will be mostly immutable and much easier to reason about.